crates/cxx-qt/src/build.rs

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! This modules contains utilities for building crates that use cxx-qt-build.
+
+#[doc(hidden)]
+// This struct should not be used directly.
+// Only use the Manifests generated by cxx-qt-build.
+pub struct Manifest {
```

Comment:
any reason for this not to be in `cxx-qt-build` ? Is this so that `cxx-qt` itself could create a manifest later as it just uses cxx_build directly ? and `cxx-qt-build` could then read the `cxx-qt` build manifest too ?

Reply:
With the `links` key, we no longer need this in cxx-qt, it's now in cxx-qt-build :)